port/zos390/omrvmem.c

```diff
@@ -1675,6 +1675,6 @@ omrvmem_release_double_mapped_region(struct OMRPortLibrary *portLibrary, void *a
 void *
 omrvmem_create_double_mapped_region(struct OMRPortLibrary *portLibrary, void* regionAddresses[], uintptr_t regionsCount, uintptr_t regionSize, uintptr_t byteAmount, struct J9PortVmemIdentifier *oldIdentifier, struct J9PortVmemIdentifier *newIdentifier, uintptr_t mode, uintptr_t pageSize, OMRMemCategory *category, void *preferredAddress)
 {
-        portLibrary->error_set_last_error(portLibrary,  errno, OMRPORT_ERROR_VMEM_NOT_SUPPORTED);
+        portLibrary->error_set_last_error(portLibrary, errno, OMRPORT_ERROR_VMEM_NOT_SUPPORTED);
         return NULL;
```

Comment:
Please indent with tabs like the rest of this file.

Reply:
Done